.mockery.yaml

```
@@ -68,3 +68,8 @@ packages:
       recursive: False
       with-expecter: true
       all: True
+  github.com/berachain/beacon-kit/mod/state-transition/pkg/core:
+    config:
+      recursive: False
```

yamllint

[warning] truthy value should be one of [false, true]

(truthy)